db/config.cc

```diff
@@ -1372,6 +1373,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , _extensions(std::move(exts))
 {
     add_tombstone_gc_extension();
+    _extensions->add_schema_extension<db::tablet_hints_extension>(db::tablet_hints_extension::NAME);
```

Comment:
This patch is not needed after the options were promoted to scylla_tables (from schema extensions)